JOSESwift/Sources/Algorithms.swift

```diff
@@ -40,6 +40,7 @@ public enum SignatureAlgorithm: String {
     case ES512
 }
 
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/airsidemobile/JOSESwift/blob/7e9896f3cc484ba08f327865cabbfb4ea34cddb8/JOSESwift/Sources/Algorithms.swift#L43"></span>Limit vertical whitespace to a single empty line. Currently 2.<br />
<code>vertical_whitespace</code> <code>Algorithms.swift:43</code></td>
    </tr>
  </tbody>
</table>